ipatests/test_integration/test_replica_promotion.py

```diff
@@ -613,3 +613,26 @@ def test_sub_ca_key_replication(self):
                                                   encoding='utf-8')
         # check for cert/key import error message
         assert self.ERR_MESS not in pki_debug_log
+
+    def test_sign_with_subca_on_replica(self):
+        master = self.master
+        replica = self.replicas[0]
+
+        SUBCA_KEY_FILE = '/etc/pki/tls/private/test_subca.key'
+        SUBCA_CRT_FILE = '/etc/pki/tls/private/test_subca.crt'
```

Comment:
These variable names are confusing.  I would probably change them to better show that they are for the test certificate, rather than the subca itself.  `TEST_(KEY|CRT)_FILE` or something like that.